adapter-config-schema/precice_adapter_config_schema.json

```
@@ -115,6 +115,41 @@
     "precice_config_file_name",
     "interfaces"
   ],
+  "additionalProperties": {
+    "title": "Solver settings",
+    "description": "Optional solver specific settings. We recommend calling the main object after the solver, e.g. 'mbdyn'",
+    "type": "object",
+    "examples": [
+      "mbdyn",
+      "issm",
+      "code_aster",
+      "tau"
+    ],
+    "additionalProperties": {
+      "oneOf": [
+        {
+          "title": "String property",
+          "description": "An optional string setting, such as the location of the solver configuration file.",
+          "type": "string",
+          "examples": [
+            "root_path",
+            "model_name",
+            "input_file_name",
+            "output_file_name"
```

TO be consistent with the `precice_config_file_path`, these could also be:

```suggestion
            "input_file_path",
            "output_file_path"
```